xbmc/windowing/windows/WinSystemWin32.h

```diff
@@ -192,6 +196,7 @@ class CWinSystemWin32 : public CWinSystemBase
   bool m_bFirstResChange = true;
   std::unique_ptr<CIRServerSuite> m_irss;
   std::vector<MONITOR_DETAILS> m_displays;
+  std::unique_ptr<CWSDiscoverySupport> m_wsd = nullptr;
```

Comment:
Shouldn't this be in CServiceManager/CServiceBroker? It's not really related to the windowing system

Reply:
CServiceManager is common code for all platforms, right?

How can I initialize only in case Win32 and avoid include Windows exclusive headers here?

Sorry, I don't know this part of the code well.

Reply:
OK moved to platform/win32/PlatformWin32.cpp